### PR TITLE
NRPT-288: Manually add mine record within a collection

### PR DESCRIPTION
**Pull request by danieltruong**

### CODE REVIEW

File: angular/projects/admin-nrpti/src/app/mines/mines-collections-add-edit/mines-collections-add-edit.component.ts [hunk 1]
```diff
@@ -441,6 +456,17 @@ export class MinesCollectionsAddEditComponent implements OnInit, OnDestroy {
     }
   }
 
+  updateRecordList(recordToAdd) {
+    // tslint:disable
+    let formArray = this.myForm.get('collectionRecords') as FormArray;
```
Comment (dhlevi) [inline angular/projects/admin-nrpti/src/app/mines/mines-collections-add-edit/mines-collections-add-edit.component.ts:461]: Should be able to get rid of the tslint:disable and make formArray a const. 
`const formArray = this.myForm.get('collectionRecords') as FormArray;`

File: angular/projects/admin-nrpti/src/app/mines/mines-collections-record-add/mines-collections-record-add.component.ts [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    } else {
+      record['recordType'] = this.myForm.get('recordType').value;
+    }
+
+    // Fields that are auto populated
+    record['issuedTo'] = {
+      type: 'Company',
+      companyName: this.mine.permittee
+    };
+    record['centroid'] = [this.mine.location.coordinates[0], this.mine.location.coordinates[1]];
```
Comment (dhlevi) [inline angular/projects/admin-nrpti/src/app/mines/mines-collections-record-add/mines-collections-record-add.component.ts:91]: `record['centroid'] = [this.mine.location.coordinates[1], this.mine.location.coordinates[0]];

Coordinates are reversed. Should be Long/Lat

File: angular/projects/admin-nrpti/src/app/mines/mines-collections-add-edit/mines-collections-add-edit.component.ts (hunk 1 above)
Comment (danieltruong) [reply angular/projects/admin-nrpti/src/app/mines/mines-collections-add-edit/mines-collections-add-edit.component.ts]: e942846

File: angular/projects/admin-nrpti/src/app/mines/mines-collections-record-add/mines-collections-record-add.component.ts (hunk 2 above)
Comment (danieltruong) [reply angular/projects/admin-nrpti/src/app/mines/mines-collections-record-add/mines-collections-record-add.component.ts]: e942846

**Pull request merged**